app/services/user-summary.js

```diff
@@ -0,0 +1,54 @@
+import Ember from 'ember';
+
+const { get, set } = Ember;
+
+export default Ember.Service.extend({
+  store: Ember.inject.service(),
+  authentication: Ember.inject.service(),
+
+  weeekEntries: null,
+  monthEntries: null,
+
+  init() {
+    this._super(...arguments);
+    set(this, 'weekEntries', []);
+    set(this, 'monthEntries', []);
+  },
+
+  currentUserId: Ember.computed.reads('authentication.userId'),
+
+  currentUserIdChanged: Ember.observer('currentUserId', function() {
+    this.reload();
+  }),
```

Comment:
Ok to let it there. Anyway move it to a component is somewhat difficult because other objects components/controllers need to talk with it in order to `reload` it.